core/controllers/base.py

```diff
@@ -156,7 +157,8 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
             user_settings = user_services.get_user_settings(
                 self.user_id, strict=False)
             if user_settings is None:
-                email = current_user_services.get_user_email(self.user)
+                email = current_user_services.get_user_email(
```

Comment:
In admin_navbar_directive.html [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/admin/admin_navbar_directive.html#L46)

and it's passed from admin.html

But the div style is display:none